MANIFEST.in

```diff
@@ -1,9 +1,10 @@
 include *.txt
 recursive-include wof/flask/templates *
-recursive-include wof/examples/flask/odm2/timeseries *
+recursive-include wof/examples *
```

Comment:
The `.py` files yes, any other file no. You will loose those `.cfg`, `.db, .`ini`, `.conf`, etc.